qemu/tests/x86_cpu_flag_disable.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+
+from provider.cpu_utils import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flags.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, boot guest with the cpu flags disabled
+    3) Check cpu flags inside guest(only for linux guest)
+    4) Check kvmclock inside guest(only for linux guest)
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    flags = params["flags"]
```

Comment:
Name it `flag` would be better, if you want it be a list, flags = params.object('flags'), but here I don't see the necessity.

Reply:
> Name it `flag` would be better, if you want it be a list, flags = params.object('flags'), but here I don't see the necessity.

Agree, update, and thanks.